Yoast/wordpress-seo
# Fix Social tabs causing crash in metabox

## Summary

This PR can be summarized in the following changelog entry:

* Fixes an issue where if you disabled either Facebook or Twitter in the Social setting, it would cause the metabox to crash on Category edit pages.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Disable either Facebook or Twitter in the Social menu.
* Edit or add a new Category. Notice that no error is displayed and tabs are disabled properly.

## Quality assurance


Fixes #9142
